strophe.jinglejs.js

```diff
@@ -71,15 +71,19 @@ var IqStanza = jxt.getDefinition('iq', 'jabber:client');
             'log:error': 'error.jingle'
          };
 
-         $.each(events, function(key, val) {
+         Object.entries(events).forEach(function(item) {
+            var key = item[0];
+            var val = item[1];
             self.manager.on(key, function() {
-               $(document).trigger(val, arguments);
+               var evt = new CustomEvent(val, arguments);
+               document.dispatchEvent(evt);
             });
          });
 
          self.manager.on('incoming', function(session) {
             session.on('change:connectionState', function(session, state) {
-               $(document).trigger('iceconnectionstatechange.jingle', [session.sid, session, state]);
+               var evt = new CustomEvent('iceconnectionstatechange.jingle', [session.sid, session, state]);
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent) the second parameter has to be a dictionary with a key `detail`.

It's not [compatible with Safari](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Browser_compatibility), but I think this is ok, because they not even support WebRTC. Maybe we should handle this case more gracefully.

Anyway this would be a breaking change, because it's not compatible with the previous jquery. Maybe we can use this as fallback if jquery is not available. What do you think?